.github/workflows/docs-check.yml

```diff
@@ -0,0 +1,20 @@
+name: Docs Check
+
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
You should run on all branches not necessary master, if the label is there, is necessary.

Reply:
I don't expect the docs labels to be used on other branches, but I suppose it doesn't hurt to allow the check to run just in case :-)